experimenter/experimenter/nimbus_ui_new/forms.py

```diff
@@ -861,14 +861,42 @@ def save_experiments_branches(self, field_name, model):
                     branch_slug=branch_slug,
                 )
 
+    def check_rollout_dirty(self, instance):
+        if not instance.is_rollout:
+            return
+
+        population_changed = (
+            "population_percent" in self.changed_data
+            and self.cleaned_data.get("population_percent")
+            != self.initial.get("population_percent")
+        )
+
+        publish_status = (
+            self.cleaned_data.get("publish_status") or instance.publish_status
```

Comment:
Oh my point was more about like, not hardcoding specific fields here but just looking at the statuses to know if we're dirty, and then using the 'disabled' flag on the fields to determine which ones are editable, becuase that's going to change over time.  But we can just leave it like this and change it later.